packages/longhorn/generated-changes/patch/README.md.patch

```diff
@@ -1,16 +1,7 @@
 --- charts-original/README.md
 +++ charts/README.md
-@@ -12,7 +12,7 @@
- 2. Longhorn Instance Manager -- Controller/replica instance lifecycle management https://github.com/longhorn/longhorn-instance-manager
- 3. Longhorn Share Manager -- NFS provisioner that exposes Longhorn volumes as ReadWriteMany volumes. https://github.com/longhorn/longhorn-share-manager
- 4. Longhorn Manager -- Longhorn orchestration, includes CSI driver for Kubernetes https://github.com/longhorn/longhorn-manager
--4. Longhorn UI -- Dashboard https://github.com/longhorn/longhorn-ui
-+5. Longhorn UI -- Dashboard https://github.com/longhorn/longhorn-ui
- 
- ## Prerequisites
- 
-@@ -21,41 +21,12 @@
- 3. Make sure `curl`, `findmnt`, `grep`, `awk` and `blkid` has been installed in all nodes of the Kubernetes cluster.
+@@ -22,41 +22,12 @@
```

Comment:
Why is there a README patch in general?

Reply:
There are **Helm** Chart installation & uninstallation guides in the original README doc. These guides don't apply to this Rancher Chart.